inclusive-design/idrc
# Add multilingual support to projects content type (resolves #627, #628, #629)

## Description

See #627 for details, and see also: https://github.com/chosww/idrc/pull/2.

## Testing

1. Check out branch.
2. Run `npm ci` to install new packages.
3. Run `npm start`.
4. Open the site and visit admin page: http://localhost:8080/admin
5. Add a French translation for a project using the editing interface and save.
6. Visit the project and confirm that you can switch back and forth between English and French.

## Related issues

Resolves #626, resolves #627, resolves #628 and resolves #629.
